pandas/tests/io/formats/test_format.py

```diff
@@ -1720,9 +1720,11 @@ def test_period(self):
                                  pd.Period('2011-03-01 09:00', freq='H'),
                                  pd.Period('2011-04', freq='M')],
                            'C': list('abcd')})
-        exp = ("        A                B  C\n0 2013-01          2011-01  a\n"
```

Comment:
Two changes (sorry)

1. newlines for readability
2. that extra space in the repr.